topography/include/gaussian_topog.inc

```diff
@@ -29,72 +29,34 @@
 
 !> @addtogroup gaussian_topog_mod
 !> @{
-module gaussian_topog_mod
 
-use  fms_mod, only: check_nml_error,                 &
-                    stdlog, write_version_number,    &
-                    mpp_pe, mpp_root_pe,             &
-                    error_mesg, FATAL
-
-use constants_mod, only: pi
-
-use mpp_mod,       only: input_nml_file
-
-implicit none
-private
+!#######################################################################
 
-public :: gaussian_topog_init, get_gaussian_topog
 
-!-----------------------------------------------------------------------
-! <NAMELIST NAME="gaussian_topog_nml">
-!   <DATA NAME="height" UNITS="meter" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     Height in meters of the Gaussian mountains.
-!    </DATA>
-!   <DATA NAME="olon, olat" UNITS="degree" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     The longitude and latitude of mountain origins (in degrees).
-!    </DATA>
-!   <DATA NAME="wlon, wlat" UNITS="degree" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     The longitude and latitude half-width of mountain tails (in degrees).
-!    </DATA>
-!   <DATA NAME="rlon, rlat" UNITS="degree" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     The longitude and latitude half-width of mountain ridges (in degrees).  For a
-!     "standard" Gaussian mountain set rlon=rlat=0.
-!    </DATA>
-!
-!    <DATA NAME="NOTE">
-!     The variables in this namelist are only used when routine
-!     <TT>gaussian_topog_init</TT> is called.  The namelist variables
-!     are dimensioned (by 10), so that multiple mountains can be generated.
+!> The height, position, width, and elongation of the mountain
```

Comment:
I think this comment block has been misplaced?

Reply:
@rem1776 told me to move it for documentation, it's right above `get_gaussian_topog` now

Reply:
it seems to be above gaussian_topog_init?

Reply:
https://github.com/NOAA-GFDL/FMS/pull/1250#discussion_r1310661855